docs/conf/helpop-full.conf.example

```diff
@@ -927,11 +930,39 @@ using their cloak when they quit.">
                     channel (requires stripcolor module).
  T                  Blocks /NOTICEs to the channel from users who are
                     not at least halfop (requires nonotice module).
-
- g <mask>           Blocks messages matching the given glob mask
-                    (requires chanfilter module).
- X <mode>           Makes channel operators immune to the specified
-                    restrictive mode (requires exemptchanops module).
+ X <mode>:<status>  Makes users of <status> or higher immune to the
```

Comment:
Mode means something different in IRC parlance. Can we pick a better word here?

Reply:
That would work, but then you'd have to realign all of the columns. How about "type"?

Reply:
Yeah, "type" would work; then I can specify them as "restriction types" in the description to be clear.  
Though they way it's done in master for m_repeat (with an off-set into the description text) doesn't seem terrible, in a compromise between space and ease.